src/incubator/f5-bigip-ctlr/templates/f5-bigip-ctlr-deploy.yaml

```diff
@@ -33,6 +33,7 @@ spec:
       containers:
       - name: {{ template "f5-bigip-ctlr.name" . }}
         image: "{{ .Values.image.user }}/{{ .Values.image.repo }}:{{ .Values.image.tag }}"
+{{- if .Values.secrets_env }}
```

Comment:
Why not preserve env vars (which is the documentation example) as the default and make a mount option conditional on the presence of a mount dir parameter? (See below)

Reply:
It's related to the following: https://support.f5.com/csp/article/K58935003